programs/cleos/main.cpp

```diff
@@ -2421,7 +2421,25 @@ CLI::callback_t header_opt_callback = [](CLI::results_t res) {
    return true;
 };
 
+
+// TODO remove this when eos-vm-oc is broken out into a separate lib and not linked in
```

Comment:
Since this is for eos-vm-oc, it would be better to put it in compile_monitor_trampoline::start.

Reply:
I considered that but I didn't want to affect signal handling in other processes and just limit it to cleos.